deposit/src/test/java/edu/unc/lib/deposit/normalize/DirectoryToBagJobTest.java

```diff
@@ -102,25 +98,24 @@ public void testConversion() throws Exception {
 		assertEquals(depositBag.size(), 1);
 		
 		Bag bagFolder = model.getBag((Resource) depositBag.iterator().next());
-		assertEquals("Bag folder label was not set", "Test File", bagFolder.getProperty(dprop(model, label)).getString());
-		assertEquals("Content model was not set", CONTAINER.toString(),
-				bagFolder.getPropertyResourceValue(fprop(model, hasModel)).getURI());
+		assertEquals("Bag folder label was not set", "Test File", bagFolder.getProperty(CdrDeposit.label).getString());
+		assertEquals("Content model was not set", RDF.Bag, bagFolder.getPropertyResourceValue(RDF.type));
```

Comment:
An empty folder should have the Cdr.Folder model.  You can test that with `resc.hasProperty(property, value)`